portability-gateway/src/main/java/org/dataportabilityproject/gateway/action/startjob/StartJobAction.java

```diff
@@ -200,8 +200,9 @@ private void encryptAndUpdateJobWithCredentials(
         asymmetricKeyGenerator.parse(
             BaseEncoding.base64Url().decode(job.jobAuthorization().authPublicKey()));
     Encrypter asymmetricEncrypter = EncrypterFactory.create(authPublicKey);
+
     String encryptedAuthSecretKey =
-        asymmetricEncrypter.encrypt(new String(authSecretKey.getEncoded(), Charsets.UTF_8));
+        asymmetricEncrypter.encrypt(BaseEncoding.base64Url().encode(authSecretKey.getEncoded()));
```

Comment:
This was it! I notice it's in the Decrypter, this may make sense in the Encrypter, I'll follow up after this PR

Reply:
OK

Reply:
Good find! Do we need the extra encoding? Is authSecretKey already encoded ("getEncoded()")? Can we remove it on both ends?